scripts/osx-create-dmg.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+APP="frontends/qt/build/osx/BitBox.app"
+INSTALLER="frontends/qt/build/osx/BitBoxApp Installer.dmg"
+
+if ! type create-dmg > /dev/null; then
+    echo "create-dmg not available. Installing with Homebrew."
+    brew install create-dmg
+fi
```

Comment:
Imo it's not very nice to mess with someone's system without prompt/confirmation. I prefer to exit with an error and instruct the user to run the command to fix it.